src/client/sandbox/index.js

```diff
@@ -198,6 +200,7 @@ export default class Sandbox extends SandboxBase {
         this.node.attach(window);
         this.upload.attach(window);
         this.cookie.attach(window);
+        this.console.attach(window);
```

Comment:
Should we reattach `ConsoleSandbox` after `document.write` is called?
See `reattach` method and `DOCUMENT_CLEANED_EVENT` event.

Reply:
It seems, we should